settings/common.py

```diff
@@ -34,19 +37,18 @@
     "facebook_app_id": os.environ.get("FACEBOOK_APP_ID", None),
 }
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
-    "django.contrib.auth.middleware.SessionAuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "simple_history.middleware.HistoryRequestMiddleware",
 )
 
 CORE_APPS = (
-    "flat",  # https://github.com/elky/django-flat-theme
+    # "flat",  # https://github.com/elky/django-flat-theme
```

Comment:
Could you please advise why we are commenting out this line?

Reply:
django flat theme is for older version of django.
![image](https://user-images.githubusercontent.com/69859505/211560225-f2c9a681-fc5e-41cf-b86c-5558d9da83d0.png)